main.py

```diff
@@ -1,11 +1,30 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
-
-
-if __name__ == "__main__":
-    main()
+    with open("players.json") as players_data:
+        players_info = json.load(players_data)
+    for player_name, player_data in players_info.items():
+        player = Player(
+            nickname=player_name,
+            email=player_data["email"],
+            bio=player_data["bio"],
+            race=Race.objects.get_or_create(
+                name=player_data["race"]["name"],
+                defaults={"description": player_data["race"]["description"]}
+            )[0],
```

Comment:
The `get_or_create` method will not update the `description` if a `Race` with the same name already exists but has a different description. Consider handling this case if updating the description is necessary.